src/strings/Strings.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace strings;
+
+class Strings implements StringsInterface
+{
+
+    /**
+     * The $input variable contains text in snake case (i.e. hello_world or this_is_home_task)
+     * Transform it into camel cased string and return (i.e. helloWorld or thisIsHomeTask)
+     * @see http://xahlee.info/comp/camelCase_vs_snake_case.html
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param string $input
+     * @return string
+     */
+    public function snakeCaseToCamelCase(string $input): string
+    {
+        $input = strtolower($input);
+        if(!str_contains($input,"_")){
+            return $input;
+        }
+        for($i=0;$i<strlen($input);$i++){
+            if($input[$i] =='_'){
+                $input[$i+1] = strtoupper($input[$i+1]);
+            }
+        }
+        return str_replace("_","",$input);
+    }
+
+    /**
+     * The $input variable contains multibyte text like 'ФЫВА олдж'
+     * Mirror each word individually and return transformed text (i.e. 'АВЫФ ждло')
+     * !!! do not change words order
+     *
+     * Make sure the next PHPDoc instructions will be added:
+     * @param string $input
+     * @return string
+     */
+    public function mirrorMultibyteString(string $input): string
```

Comment:
Please try to simplify it.
In my opinion, there should only be 3 steps.
1. explode
2. reverse
3. implode

Reply:
![image](https://user-images.githubusercontent.com/70289785/185147290-f9a47e0d-79a4-4f6a-8c15-a7a216ded13a.png)
Then the result on the screenshot

Reply:
But I have an error with strrev, because there is a problem with encoding. So I decided to do more steps.

Reply:
OK. Let it be. Just fix the quotes here